Waarschuwing over ontbrekende onderdelen

kdekooter opened the pull request.

kdekooter committed:
Waarschuwing over ontbrekende onderdelen

kwoot commented:
Zou je deze aub ook willen indienen op de repo: OperatieBRP/OperatieBRP ? Dank!

kdekooter commented:
Zeker. Wat is de status van die fork?

kwoot committed:
Merge pull request #1 from kdekooter/master

Waarschuwing over ontbrekende onderdelen

kwoot commented:
Mijn poging om er leven in te blazen.

Commit:
We found a potential security vulnerability in one of your dependencies.

The moment dependency defined in package.json has a known high severity
security vulnerability in version range < 2.11.2 and should be updated.

Commit:
Merge branch 'master' of https://github.com/OperatieBRP/OperatieBRP

kdekooter committed:
Merge pull request #1 from OperatieBRP/master

Merging security vulnerability for moment.js

bvhme commented:
Ik ga deze pull request sluiten als voorbereiding om de codebase als 'gearchiveerd' te kunnen aanmerken. Helaas kan ik op dit moment niet op de inhoud ingaan.
